### PR TITLE
sparse/src: Work around gnu compiler bug

### PR DESCRIPTION
**Pull request by e10harvey**
@vqd8a reported that gcc 8 is erroring out here (thanks for reporting this, @vqd8a!). This is a known gnu bug: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=81429.

Fixes #1862.

### CODE REVIEW

**Review by brian-kelley — APPROVED** [review]

**Review by vqd8a — APPROVED** [review]
@e10harvey Thanks for fixing it.

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pull Request AutoTester' - Error: Jenkins Jobs - A user has pushed a change to the PR before testing completed. NEW EVENT 'committed', ID C_kwDOBK7s5toAKDM2MmJlNWU3MGNhNzBjMWM1ZWQ5NzE1YzBkNWE2ZDcyZWE2MDU0ZGM... The Jenkins Jobs will be shutdown; Testing of this PR must occur again.

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pre-Test Inspection' - Auto Inspected - Inspection is Not Necessary for this Pull Request.

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pull Request AutoTester' - Testing Jenkins Projects:

<details>
<summary>
  <b>Pull Request Auto Testing STARTING</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_CUDA11_CUDA11_LayoutRight</h3>
<ul>
  <li>Build Num: 685</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f75ec31ce4faee34bc2a788bf1aab021a72f6616</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ce8bb989f2100cafafd4d9529203c00209ca57e1</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>CompilerIssue</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1862</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GCC930_Light_Tpls_GCC930_Tpls_CLANG13CUDA10</h3>
<ul>
  <li>Build Num: 276</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f75ec31ce4faee34bc2a788bf1aab021a72f6616</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ce8bb989f2100cafafd4d9529203c00209ca57e1</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>CompilerIssue</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1862</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GCC1020</h3>
<ul>
  <li>Build Num: 299</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f75ec31ce4faee34bc2a788bf1aab021a72f6616</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ce8bb989f2100cafafd4d9529203c00209ca57e1</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>CompilerIssue</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1862</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GCC1020_Light_LayoutRight</h3>
<ul>
  <li>Build Num: 569</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f75ec31ce4faee34bc2a788bf1aab021a72f6616</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ce8bb989f2100cafafd4d9529203c00209ca57e1</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>CompilerIssue</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1862</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_GCC1020</h3>
<ul>
  <li>Build Num: 530</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f75ec31ce4faee34bc2a788bf1aab021a72f6616</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ce8bb989f2100cafafd4d9529203c00209ca57e1</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>CompilerIssue</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1862</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_INTEL19</h3>
<ul>
  <li>Build Num: 619</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f75ec31ce4faee34bc2a788bf1aab021a72f6616</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ce8bb989f2100cafafd4d9529203c00209ca57e1</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>CompilerIssue</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1862</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_CLANG1001</h3>
<ul>
  <li>Build Num: 672</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f75ec31ce4faee34bc2a788bf1aab021a72f6616</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ce8bb989f2100cafafd4d9529203c00209ca57e1</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>CompilerIssue</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1862</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_A64FX_Tpls_ARMPL2110</h3>
<ul>
  <li>Build Num: 473</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f75ec31ce4faee34bc2a788bf1aab021a72f6616</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ce8bb989f2100cafafd4d9529203c00209ca57e1</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>CompilerIssue</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1862</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_A64FX_GCC1020</h3>
<ul>
  <li>Build Num: 468</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f75ec31ce4faee34bc2a788bf1aab021a72f6616</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ce8bb989f2100cafafd4d9529203c00209ca57e1</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>CompilerIssue</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1862</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_VEGA908_ROCM520</h3>
<ul>
  <li>Build Num: 468</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f75ec31ce4faee34bc2a788bf1aab021a72f6616</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ce8bb989f2100cafafd4d9529203c00209ca57e1</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>CompilerIssue</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1862</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Using Repos:</h2>

<dl>
  <dt>Repo: KOKKOSKERNELS (e10harvey/kokkos-kernels)</dt>
  <ul>
    <li>Branch: workaround_gnu_bug_81429</li>
    <li>SHA: f75ec31ce4faee34bc2a788bf1aab021a72f6616</li>
    <li>Mode: TEST_REPO</li>
  </ul>
</dl>

Pull Request Author: e10harvey
</details>

File: sparse/src/KokkosSparse_BsrMatrix.hpp [hunk 1]
```diff
@@ -625,25 +625,31 @@ class BsrMatrix {
...
+/// \param label
+/// \param nrows [in] The number of rows.
+/// \param ncols [in] The number of columns.
+/// \param annz  [in] Filler for annz.
+/// \param vals [in/out] The entries.
+/// \param rows [in/out] The row map (containing the offsets to the
+///   data in each row).
+/// \param cols [in/out] The column indices.
+/// \param blockDimIn [in] The block dimensions.
+#if defined(DOXY)
```
Comment (ndellingwood) [inline sparse/src/KokkosSparse_BsrMatrix.hpp:643]: @e10harvey just to clarify the use of the guard here for the fix - apparently the attribute must proceed the argument for doxygen? And generally this should be supported, but due to compiler bug with gcc pre-9.3.0 (which does not properly support the use of attribute preceding the argument) instead for all cases the attribute follows the argument (which bypasses the compiler bug)? 
I had expected the guard to involve a check on the compiler and version, so I'm just trying to understand and untangle the connection with DOXY here

Comment (e10harvey) [reply sparse/src/KokkosSparse_BsrMatrix.hpp:643]: My understanding is that for doxygen < 1.9.7, we need the attribute to be at the beginning. For GNU 8, we need the attribute to be at the end due to a compiler bug. The CI checks are installing the latest doxygen available in the CI VM, which is 1.9.6. Note that doxygen 1.9.7 has been released, though.

**Review by ndellingwood — APPROVED** [review]

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pull Request AutoTester' - Jenkins Testing: all Jobs PASSED

<details>
<summary>
  <b>Pull Request Auto Testing has PASSED</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_CUDA11_CUDA11_LayoutRight</h3>
<ul>
  <li>Build Num: 685</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f75ec31ce4faee34bc2a788bf1aab021a72f6616</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ce8bb989f2100cafafd4d9529203c00209ca57e1</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>CompilerIssue</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1862</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GCC930_Light_Tpls_GCC930_Tpls_CLANG13CUDA10</h3>
<ul>
  <li>Build Num: 276</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f75ec31ce4faee34bc2a788bf1aab021a72f6616</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ce8bb989f2100cafafd4d9529203c00209ca57e1</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>CompilerIssue</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1862</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GCC1020</h3>
<ul>
  <li>Build Num: 299</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f75ec31ce4faee34bc2a788bf1aab021a72f6616</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ce8bb989f2100cafafd4d9529203c00209ca57e1</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>CompilerIssue</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1862</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GCC1020_Light_LayoutRight</h3>
<ul>
  <li>Build Num: 569</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f75ec31ce4faee34bc2a788bf1aab021a72f6616</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ce8bb989f2100cafafd4d9529203c00209ca57e1</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>CompilerIssue</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1862</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_GCC1020</h3>
<ul>
  <li>Build Num: 530</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f75ec31ce4faee34bc2a788bf1aab021a72f6616</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ce8bb989f2100cafafd4d9529203c00209ca57e1</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>CompilerIssue</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1862</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_INTEL19</h3>
<ul>
  <li>Build Num: 619</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f75ec31ce4faee34bc2a788bf1aab021a72f6616</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ce8bb989f2100cafafd4d9529203c00209ca57e1</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>CompilerIssue</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1862</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_CLANG1001</h3>
<ul>
  <li>Build Num: 672</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f75ec31ce4faee34bc2a788bf1aab021a72f6616</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ce8bb989f2100cafafd4d9529203c00209ca57e1</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>CompilerIssue</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1862</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_A64FX_Tpls_ARMPL2110</h3>
<ul>
  <li>Build Num: 473</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f75ec31ce4faee34bc2a788bf1aab021a72f6616</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ce8bb989f2100cafafd4d9529203c00209ca57e1</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>CompilerIssue</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1862</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_A64FX_GCC1020</h3>
<ul>
  <li>Build Num: 468</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f75ec31ce4faee34bc2a788bf1aab021a72f6616</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ce8bb989f2100cafafd4d9529203c00209ca57e1</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>CompilerIssue</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1862</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_VEGA908_ROCM520</h3>
<ul>
  <li>Build Num: 468</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>f75ec31ce4faee34bc2a788bf1aab021a72f6616</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>ce8bb989f2100cafafd4d9529203c00209ca57e1</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>CompilerIssue</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1862</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>
</details>

**Comment by ndellingwood** [comment]
The github-OSX / osx-ci (SERIAL, Debug, OFF) build cancelled (I'm assuming maybe timeout waiting to launch?), is there any need to relaunch that test or is the testing coverage sufficient that we merge this PR (all AT checks passed)?

**Pull request merged**